tests/unit/amo/components/TestUserProfile.js

```diff
@@ -411,9 +411,11 @@ describe(__filename, () => {
   });
 
   it("renders the user's biography", () => {
+    const biography = '<blockquote><b>Not even vegan!</b></blockquote>';
```

Comment:
Note that I cannot reproduce the issue in the test case here, because it is a server vs client issue.